test/drop_test/event.py

```diff
@@ -27,71 +28,56 @@ def get_uuid(slot: int) -> str:
     return uuid
 
 
+def compare(s, t):
+    t = list(t)  # make a mutable copy
+    try:
+        for elem in s:
+            t.remove(elem)
+    except ValueError:
+        return False
+    return not t
+
+
```

Comment:
`is_equal` could be slight improvement on naming

Reply:
Done